Bottom: 07ccf4ee25d2f6927c1bde0ad8beb85ca59f99a2
Top:    ece73d56df331fd0c13dd1c61da4ae950001ac75
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 10:31:42 -0500

Use internal mail server


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index 7f2ff830..9683a4fc 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -54,10 +54,10 @@ mn = reporters.MailNotifier(
     sendToInterestedUsers=True,
     extraRecipients=["andrew.cooper@novatechweb.com"],
     useTls=True,
-    relayhost=os.getenv('BUILDBOT_SMTP_HOST',"relay.dnsexit.com"),
-    smtpPort=26,
-    smtpUser=os.getenv('BUILDBOT_SMTP_USER', "Dean.cox"),
-    smtpPassword=os.getenv('BUILDBOT_SMTP_PASS',"no5156")
+    relayhost=os.getenv('BUILDBOT_SMTP_HOST'),
+    smtpPort=os.getenv('BUILDBOT_SMTP_PORT'),
+    smtpUser=os.getenv('BUILDBOT_SMTP_USER'),
+    smtpPassword=os.getenv('BUILDBOT_SMTP_PASS')
     )
 c['services'].append(mn)
 
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index d261448d..43907def 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -38,9 +38,10 @@ container_env:
   BUILDBOT_IRC_HOST: 'chat.freenode.net'
   BUILDBOT_IRC_NICK: 'NovaTechCI'
   BUILDBOT_IRC_PASS: '{{ buildbot_irc_password }}'
-  BUILDBOT_SMTP_HOST: 'relay.dnsexit.com'
-  BUILDBOT_SMTP_USER: 'Dean.cox'
-  BUILDBOT_SMTP_PASS: 'no5156'
+  BUILDBOT_SMTP_HOST: '{{ exim4_hostname }}'
+  BUILDBOT_SMTP_PORT: '587'
+  BUILDBOT_SMTP_USER: 'buildbot'
+  BUILDBOT_SMTP_PASS: '{{ buildbot_email_password }}'
   BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
   BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
